django/db/models/fields/__init__.py

```diff
@@ -2100,6 +2131,15 @@ def formfield(self, **kwargs):
         defaults.update(kwargs)
         return super(PositiveIntegerField, self).formfield(**defaults)
 
+    def rel_db_type(self, connection):
```

Comment:
Could there be any value in a mixin for this method since it's shared between the PositiveInteger fields?